docs/source/use_cases/index.rst

```diff
@@ -0,0 +1,25 @@
+Use case scenarios
```

Comment:
Should the use-cases be presented at the docs? I think it would be nice to place this stuff at some kind of blog posts.
Imo it's better to write section about where is the technology is applicable and where isn't, e.g:
`It probably will have no profit if you are trying to build centralized application of one component. Though the main features of DLT comes when there is a multiple untrusted parties are going to make some stuff`